transform_to_tf/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+cmake_minimum_required(VERSION 2.8.0)
+project(transform_to_tf)
+
+add_definitions(-std=c++11)
+
+find_package(catkin_simple REQUIRED)
+catkin_simple(ALL_DEPS_REQUIRED)
+
+find_package(Eigen REQUIRED)
```

Comment:
I have no good answer for that other than to be consistent with most of our other packages which seem to use the system lib... My preference is to depend only catkinized packages.